lib/chef/resource/sysctl.rb

```diff
@@ -45,6 +45,10 @@ class Sysctl < Chef::Resource
         coerce: proc { |v| coerce_value(v) },
         required: true
 
+      property :comment, [Array, String],
+        description: "Comments, placed above the resource setting in the generated file. For multi-line comments, use an array of strings, one per line.",
```

Comment:
Why an array of strings vs a single multi-line string?

Reply:
Simplicity and ease of use - prepending '#' to the beginning of each entry in an array is a simple task. I'm sure it's quite trivial for multi-line strings too, but this to me is more readable.
Personal preference, admittedly.